packages/paste-core/components/menu/src/constants.ts

```diff
@@ -2,3 +2,5 @@ export const MenuItemVariants = {
   DEFAULT: 'default',
   GROUP_ITEM: 'group_item',
 } as const;
+
+export const DEFAULT_ELEMENT_NAME = 'MENU';
```

Comment:
This is a good idea!

Reply:
I'm not sure this provides that much value personally, since the default element name is just a copy of the component and the string concatenation adds developer complexity.

Reply:
I was thinking that it would be useful if we ever decided to change the default element name, since we'd only have to make that change in 1 place. I guess maybe that's not going to be too common? I followed a similar pattern in Toast (adding the default element name to the constants folder), but open to not doing it that way.